uw_gws/__init__.py

```diff
@@ -295,42 +296,42 @@ def _valid_group_id(self, group_id):
         if (group_id is None or not self.RE_GROUP_ID.match(group_id)):
             raise InvalidGroupID(group_id)
 
-    def _get_resource(self, url, headers={}):
-        headers["Accept"] = "application/json"
-        if self.actas:
-            headers["X-UW-Act-as"] = self.actas
-
-        response = self.DAO.getURL(url, headers)
+    def _get_resource(self, url):
```

Comment:
Should we add a comment or rename the method here to signify that this should be used to get any resource except for effective membership?

Reply:
Comment added.